workflow/flow.cylc

```diff
@@ -17,7 +17,7 @@
         """
         P1D = """
             get_all_overpass_times[^] => get_single_overpass_time<satellite>
-            INIT[^]:succeed-all => LOAD:succeed-all => PREPROCESS:succeed-all => exportH5 => done
+            INIT[^]:succeed-all => LOAD:succeed-all => PREPROCESS:succeed-all => exportH5<satellite> => done
```

Comment:
So was the exportH5 getting lost on where to save results since it wasn't getting the satellite info?

Reply:
Also, on line 24 -- is it still the case that if preprocessing fails for one of the list of images, tracking won't get applied? Any progress/thoughts toward how to handle cases where no labeled image is created?

Reply:
The problem was that a task named `exportH5` doesn't exist. `exportH5<satellite>` is defined, but it wasn't being referenced properly in the graph. It was treating `exportH5` as a dummy task and not running any code (but also not crashing).

Reply:
This PR only fixes the missing exportH5s. 
You're still correct that right now, if a task fails, all the subsequent tasks in the graph fail.

Reply:
I've added a comment about the failing preprocessing to the issue there: https://github.com/WilhelmusLab/ice-floe-tracker-pipeline/issues/212#issuecomment-2608146153

Reply:
Oh I see, sorry, I totally misnamed this PR.